Kitodo/src/main/webapp/WEB-INF/templates/includes/processEdit/details.xhtml

```diff
@@ -61,12 +61,13 @@
                 </p:selectOneMenu>
             </div>
 
-            <div>
-                <p:outputLabel for="isTemplate" value="#{msgs.istTemplate}"/>
-                <p:selectBooleanCheckbox id="isTemplate" value="#{ProzessverwaltungForm.process.template}">
-                    <p:ajax event="change" oncomplete="jQuery('#editForm\\:saveButtonToggler').click()"/>
-                </p:selectBooleanCheckbox>
-            </div>
+            <!-- TODO: decide whether the following switch is required -->
```

Comment:
Maybe template handling should be independent of process handling. They are similar but not equal in every aspect.

Reply:
Yes, that is the reasoning why the "template" switch has been removed from the `processEdit` page for now. I outcommented the code instead of removing it entirely, since the decision whether this functionality will still be required in the future or not has not been made, yet.